tests/attestation/primary.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:latest
```

Comment:
the image I've been using is: mcr.microsoft.com/cbl-mariner/base/python:3.9 . worth a try to see if it works imo

Reply:
I tried it initially and unfortunately it didn't work out of the box, but I'm sure if can be made to work with some adjustments

Reply:
Couldn't easily figure this out so I suggest we merge as is for now